src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -41,4 +41,35 @@ public void arcadeDrive(double move, double rot) {
   public void tankDrive(double left, double right) {
     m_drive.tankDrive(left, right);
   }
+
+  public double getDistance() {
+    // Calculates the average wheel sensor position(for example, if the right wheels did a one full rotation, and the left wheels did a 50% rotation, averageWheels will be 0.75 rotation)
+    double averageWheels = (m_leftFront.getSelectedSensorPosition() + m_leftRear.getSelectedSensorPosition() + m_rightFront.getSelectedSensorPosition() + m_rightRear.getSelectedSensorPosition()) / 4 / DriveTrainConstants.kMotorToWheelRatio / DriveTrainConstants.kEncoderResolution; 
```

Comment:
This lines are way too complicated, suggesting separating calculations into several commands.

```suggestion
    double leftAvgDistance = (leftFront.get...() + leftRear.get...()) / 2;
    double rightAvgDistance = ...;  // same but for right
    double centralAvg = (leftAvg + rightAvg) / 2;
    // and here transform centralAvg to the units we want
```